redfish/config.py

```diff
@@ -43,14 +58,13 @@ def initialize_logger(REDFISH_LOGFILE,
     formatter = logging.Formatter(
         '%(asctime)s :: %(levelname)s :: %(message)s'
         )
+    f = open(os.path.expandvars(REDFISH_LOGFILE), 'w')
+    f.close()
+
```

Comment:
Without this, in case the file doesn't exist yet, it's not created by RotatingFileHandler. I experienced that thus added these 2 lines. I'm not sure I undertand why we need a with here, but let's discuss this afternoon.